src/platform/scancode.rs

```diff
@@ -0,0 +1,31 @@
+#![cfg(any(
+  target_os = "windows",
+  target_os = "macos",
+  target_os = "linux",
+  target_os = "dragonfly",
+  target_os = "freebsd",
+  target_os = "netbsd",
+  target_os = "openbsd"
+))]
+
+// TODO: Maybe merge this with `modifier_supplement` if the two are indeed supported on the same
+// set of platforms
```

Comment:
Yeah, I think it would be a lot better to merge them. Maybe also `global_shortcut`.
Does an integrated module call `shortcut` or `accelerator` make sense?

Reply:
I'm not sure I understand this right; `Does an integrated module call shortcut or accelerator make sense?`